### PR TITLE
changed format to fstring literals

### PR DESCRIPTION
**Pull request by yukilikespie**

### CODE REVIEW

File: tor_core/initialize.py [hunk 1]
```diff
@@ -197,19 +193,15 @@ def populate_subreddit_lists(config):
         config.subreddits_domain_filter_bypass
     )
     logging.debug(
-        'Retrieved subreddits that bypass the domain filter: {}'.format(
-            config.subreddits_domain_filter_bypass
-        )
+        f'Retrieved subreddits that bypass the domain filter: {config.subreddits_domain_filter_bypass}'
```
Comment (itsthejoker) [inline tor_core/initialize.py]: I just realized that we don't actually have code standards in this repo. While I fix that, could you make sure that none of the lines changed in these PRs are longer than 80 characters? Archaic, I know, but it really does make a difference when viewing multiple files on the same screen.

Comment (perryprog) [reply tor_core/initialize.py]: Gosh darn it @itsthejoker that's what I was doing

Comment (yukilikespie) [reply tor_core/initialize.py]: @itsthejoker ok, sure

**Review by perryprog — APPROVED** [review]

**Review by TheLonelyGhost — APPROVED** [review]
LGTM! 👍

**Comment by TheLonelyGhost** [comment]
I hopped in there and fixed the 80 character line length thing for you. Tedious work and I was already pulling the repo down locally to look at it. 😄 

Great work, thank you so much, @yukilikespie!

**Pull request merged**